pkg/cmd/auth/shared/login_flow_test.go

```diff
@@ -119,6 +119,72 @@ func TestLogin_ssh(t *testing.T) {
 	assert.Equal(t, "ssh", cfg["example.com:git_protocol"])
 }
 
+func TestLogin_noSSHPrompt(t *testing.T) {
+	dir := t.TempDir()
+	ios, _, stdout, stderr := iostreams.Test()
+
+	tr := httpmock.Registry{}
+	defer tr.Verify(t)
+
+	tr.Register(
+		httpmock.REST("GET", "api/v3/"),
+		httpmock.ScopesResponder("repo,read:org"))
+	tr.Register(
+		httpmock.GraphQL(`query UserCurrent\b`),
+		httpmock.StringResponse(`{"data":{"viewer":{ "login": "monalisa" }}}`))
```

Comment:
**Is it valuable refactoring some of the other tests into a test table approach to reduce some of the duplicate code?**

There is nothing technically wrong with this new test; merely wondering the value in consolidating logic makes it more or less managable.

Reply:
Thanks for this. I pushed another commit to refactor those into a single table test. Also added one more test case.